src/experiments.js

```diff
@@ -47,6 +51,102 @@ export function isCanary(win) {
 }
 
 
+function getExperimentsKey_() {
+  let key;
+  const url = '' //TODO(kmh287): fill this in
```

Comment:
I don't think we want to do that since it'll prevent offline functionality. We can patch prod if we really need to kill a feature, though I don't think that will be needed in practice.